tests-e2e/cypress/services/sdc-collection-api-service.ts

```diff
@@ -681,6 +722,49 @@ export class SdcCollectionApiService {
     return ells;
   }
 
+  async deleteExistingTestData(schools: SchoolEntity[]) {
+    const districtIds = new Set();
+
+    try {
+      const deleteSchoolPromises: Promise<void>[] = [];
+      schools.forEach((school) => {
+        const schoolId = school.schoolId;
+        districtIds.add(school.districtId);
+        const urlGetActiveSdcSchoolCollection = `${this.config.env.studentDataCollection.base_url}${SDC_COLLECTION_SEARCH_ENDPOINT}/${schoolId}`;
+        const deletePromise = this.restUtils.getData<SdcSchoolCollection>(urlGetActiveSdcSchoolCollection).then(async (activeSchoolSdcCollection) => {
+          await this.restUtils.deleteData(`${this.config.env.studentDataCollection.base_url}${SDC_COLLECTION_ENDPOINT}/${activeSchoolSdcCollection.sdcSchoolCollectionID}`);
+        });
+        deleteSchoolPromises.push(deletePromise);
+      });
+      await Promise.all(deleteSchoolPromises);
```

Comment:
@eckermania Need to call the new endpoint to get then delete sdcSchoolCollections here